machine-learning/app/test_main.py

```diff
@@ -90,6 +90,7 @@ def test_sets_default_provider_options(self, mocker: MockerFixture) -> None:
         assert encoder.provider_options == [
             {"device_type": "GPU_FP32"},
             {"arena_extend_strategy": "kSameAsRequested"},
+            {"cache_dir": f"{encoder.cache_dir}/openvino"},
```

Comment:
This should go in the first dictionary instead of being a new entry

Reply:
right, I completely missed that